### PR TITLE
Sync bugfixing

### PR DESCRIPTION
**Pull request by dsp0011**

### CODE REVIEW

**Comment by dsp0011** [comment]
For some reason it re-added python files from last commit, so I had to remove them manually

File: src/lib/components/NewLayeredMap/components/CompositeMapLayers.js [hunk 1]
```diff
@@ -279,13 +280,18 @@ class CompositeMapLayers extends Component {
                 this.props.syncDrawings ? this.addItemToLayer(item, this.context.drawLayer, false) : itemsToDraw[item.type] = item;
             }   
         }
-
-        for (const item of this.context.drawLayerData) {
-            if (!itemsToDraw[item.type] || itemsToDraw[item.type].creationTime < item.creationTime) {
-                this.addItemToLayer(item, this.context.drawLayer, false);
-                delete itemsToDraw[item.type]
+        console.log(itemsToDraw)
```
Comment (eolweus) [inline src/lib/components/NewLayeredMap/components/CompositeMapLayers.js:283]: remove console.logs from composite map layers

File: src/lib/components/NewLayeredMap/components/Controls/components/DrawControls.js [hunk 2]
```diff
@@ -120,16 +129,20 @@ class DrawControls extends Component {
             }
             const d = new Date();
             newLayer["creationTime"] = d.getTime();
-            props.syncDrawings ? this.context.syncedDrawLayerAdd([newLayer]) : this.context.drawLayerAdd([newLayer]);
+            // TODO: create ref to read syncDrawings    
+            syncDrawings && (console.log("syncDraw is enabled"))
```
Comment (eolweus) [inline src/lib/components/NewLayeredMap/components/Controls/components/DrawControls.js:133]: Remove console.logs from DrawControls

File: src/lib/components/NewLayeredMap/NewLayeredMap.jsx [hunk 3]
```diff
@@ -87,8 +87,8 @@ class NewLayeredMap extends Component {
         }
     }
 
-    componentDidUpdate() {
-        
+    componentDidUpdate(prevProps) {
+ 
     }
```
Comment (eolweus) [inline src/lib/components/NewLayeredMap/NewLayeredMap.jsx:92]: Do we need this?

**Pull request merged**